1/PumpForge.md

```diff
@@ -0,0 +1,62 @@
+🚨 **Please don't write in this file**
+
+Don't touch this file ----- please create a copy and name it after your project
+
+# PumpForge
+
+## Link to your project's GitHub repository 
+
+- [https://github.com/lanacreates/PumpForge](https://github.com/lanacreates/PumpForge)
+
+## Team Members GitHub username
+
+- @lanacreates
+
+## Former Participation in Celo Hackathons
+
+
+## Monthly Goal for this Proof of Ship
+
+**Monthly Goal (Feb 16th - Feb 28th):**  
+I plan to build the core smart contract that enables users to launch their own tokens with a fixed supply of 1 billion coins, set custom metadata (name, symbol, and image via IPFS), and integrate initial swap functionality for trading Celo or cUSD for these tokens.
+
+**Weekly Milestones:**
+- **Week 1:**  
+  - Set up development environment (Hardhat/Truffle, Node.js, Celo CLI) and initialize our GitHub repository.
```

Comment:
Have you considered using the [Celo Composer](https://github.com/celo-org/celo-composer) for setting up the project? It doesn't come with the node.js part, but that can be easily added. Make sure to use the MiniPay template. 

What are you planning to use the Celo CLI for? It is mostly used by Validators, for node operations.